docs/dapp/opl/setup.md

```diff
@@ -15,10 +15,21 @@ Our smart contracts will live inside a Hardhat project under
 `opl-secret-ballot/backend`, and our VueJS app will be in the
 `opl-secret-ballot/frontend`.
 
+## Workspace
+
+We suggest using [`pnpm`](https://pnpm.io/motivation), and creating a workspace
+file `opl-secret-ballot/pnpm-workspace.yaml` with the following content:
+
+```yaml
+packages: [frontend, backend]
+```
+
 ## Hardhat
 
 Let's create a new Hardhat project.
 
+_Note_: we are compatible with Hardhat up to `2.12.6`.
```

Comment:
```suggestion
:::info

Currently, we are compatible with Hardhat up to `2.12.6`.

:::
```

Reply:
I should also note technically it also includes `2.12.7`.